corehq/apps/domain/forms.py

```diff
@@ -402,6 +403,12 @@ def clean_sub_area(self):
 
 class DomainGlobalSettingsForm(forms.Form):
     hr_name = forms.CharField(label=_("Project Name"))
+
+    alias = forms.CharField(
+        required=False,
+        label=_("Project Alias"),
```

Comment:
this should be `ugettext_lazy` or `ugettext_noop` since it's defined outside the scope of a user/session